lib/Listener/LoadAdditionalListener.php

```diff
@@ -10,23 +10,36 @@
 
 use OCA\EndToEndEncryption\AppInfo\Application;
 use OCA\Files\Event\LoadAdditionalScriptsEvent;
+use OCP\AppFramework\Services\IInitialState;
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
+use OCP\IConfig;
 use OCP\Util;
 
 /**
  * @template-implements IEventListener<LoadAdditionalScriptsEvent>
  */
 class LoadAdditionalListener implements IEventListener {
 
-	public function __construct() {
+	public function __construct(
+		private IInitialState $initialState,
+		private IConfig $config,
+		private ?string $userId,
+	) {
 	}
 
 	public function handle(Event $event): void {
 		if (!($event instanceof LoadAdditionalScriptsEvent)) {
 			return;
 		}
 
+		$this->initialState->provideInitialState(
+			'userConfig',
+			[
+				'e2eeInBrowserEnabled' => $this->config->getUserValue($this->userId, 'end_to_end_encryption', 'e2eeInBrowserEnabled', 'false') === 'true',
+			]
+		);
+
```

Comment:
I would prefer a capability :)

Reply:
That a user setting, not a capability of the server, no?

Reply:
Some capabilities (like theming) changes per user :thinking: 
Maybe it is an antipattern :thinking:

Reply:
provisioning_api too https://github.com/nextcloud/server/blob/381077028adf388a7081cf42026570c6be47b198/apps/provisioning_api/lib/Capabilities.php#L32